cpa/policy_test.go

```diff
@@ -497,3 +497,28 @@ func TestPolicyRuntimeError(t *testing.T) {
 		decision.Reason,
 	)
 }
+
+func TestEnableHard(t *testing.T) {
```

Comment:
We do not need this test, it is satisfied by the one you are doing below!
We are dog fooding the policy-agent to the testrunner and so we don't need to write go tests for policy evaluations any more.

Reply:
removed!